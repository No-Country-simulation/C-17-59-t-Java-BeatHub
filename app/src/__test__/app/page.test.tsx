import Home from "@/app/page";
import { render, screen, within } from "@testing-library/react";
import { expect, test } from "vitest";

test("render home page", () => {
  render(<Home />);
  const main = within(screen.getByRole("main"));
  expect(
    main.getByRole("heading", {
      level: 1,
      name: "BeatHub",
    })
  ).toBeDefined();
});

test("render home page to same of the snapshot", () => {
  const { container } = render(<Home />);
  expect(container).toMatchSnapshot();
});
