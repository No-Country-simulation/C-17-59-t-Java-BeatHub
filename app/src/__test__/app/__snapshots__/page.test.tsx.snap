// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`render home page to same of the snapshot 1`] = `
<div>
  <main
    class="flex min-h-screen flex-col items-center justify-between p-24"
  >
    <div>
      <h1>
        BeatHub
      </h1>
      <p>
        Ahora si bro a trabajar se ha dichooo!
      </p>
    </div>
  </main>
</div>
`;
